docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: '3'
+services:
+  db:
+    image: postgres:15-alpine
+    container_name: ciabot_postgres
+    restart: always
+    environment:
+      POSTGRES_DB: ${POSTGRES_DB}
+      POSTGRES_USER: ${POSTGRES_USER}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+    volumes:
+      - pgdata:/var/lib/postgresql/data
+    networks:
+      - ciabot_network
+    ports:
+      - "5432:5432"
+
+  ciabot:
+    build: .
+    container_name: ciabot_public
+    restart: always
+    environment:
+      POSTGRES_DB: ${POSTGRES_DB}
+      POSTGRES_USER: ${POSTGRES_USER}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+      POSTGRES_HOST: db
+    networks:
+      - ciabot_network
+    depends_on:
+      - db
+    ports:
+      - "81:3001"
```

Comment:
What's the port for?

Reply:
port is for the bot itself, i just incremented one up of the default of 80/3000.

Reply:
okay I might look into it later since I'm not sure if we need to expose container ports atm